### PR TITLE
fix tExecutor test on Windows

### PR DESCRIPTION
**Pull request by xvallspl**

### CODE REVIEW

**Comment by xvallspl** [comment]
@bellenot Would this be enough? to deal with https://github.com/root-project/roottest/pull/636

**Comment by bellenot** [comment]
> @bellenot Would this be enough? to deal with #636

I think so (it's exactly what I wanted to try). I'll approve it as soon as I managed to successfully run it on Windows

File: root/multicore/CMakeLists.txt [hunk 1]
```diff
@@ -87,7 +87,11 @@ if(ROOT_imt_FOUND)
                     DEPENDS ${GENERATE_EXECUTABLE_TEST})
 endif()
 
-ROOTTEST_GENERATE_EXECUTABLE(tExecutor tExecutor.cxx LIBRARIES Core Imt MultiProc Net TreePlayer RIO Hist MathCore)
+if(NOT MSVC OR win_broken_tests)
```
Comment (bellenot) [inline root/multicore/CMakeLists.txt:90]: ``

File: root/multicore/tExecutor.cxx [hunk 2]
```diff
@@ -25,10 +25,12 @@ int main() {
    }
 #endif
 
+#ifndef R__WIN32
```
Comment (bellenot) [inline root/multicore/tExecutor.cxx:28]:
```suggestion
#ifndef _MSC_VER
```

File: root/multicore/tExecutor.cxx [hunk 3]
```diff
@@ -43,8 +45,10 @@ int main() {
    res = ExecutorTest(ex1);
    if(res) return res+offset+4;
 #endif
+#ifndef R__WIN32
```
Comment (bellenot) [inline root/multicore/tExecutor.cxx:48]:
```suggestion
#ifndef _MSC_VER
```

**Review by bellenot — CHANGES_REQUESTED** [review]
In general `_MSC_VER` is preferred over `R__WIN32` which can also be defined when using different compilers supporting more features than MSVC

**Review by bellenot — APPROVED** [review]
Thanks!

**Pull request merged**